### ISSUE TITLE
Shotcut Not Responding after 85% of clicks

### ISSUE THREAD

**Issue by Zonenshine** [mltframework/shotcut#1190]
Beginning within the last few days after approximately 85% clicks/actions Shotcut is entering Not Responding status for about 30 seconds.  Details follow.

Version of Shotcut: Began with 21.09.20 and has persisted after upgrade to 21.10.31.  All versions are Windows 64 bit.

System specs: Lenovo ThinkPad L380 Yoga
Processor	Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz   2.71 GHz
Installed RAM	16.0 GB (15.9 GB usable)
System type	64-bit operating system, x64-based processor
Pen and touch	Pen and touch support with 10 touch points
Storage  SSD drive with 100 GB Free
OS   Windows Professional 64 bit

The problem occurs across all projects including those created in the last few weeks with version 21.09.20, and those created previously on older versions but updated successfully with 21.09.20.  All projects are 6 minutes or less and use a maximum of seven tracks.  

I have run Task Manager while experiencing this problem and CPU and Memory do not go above 40% unless exporting video.

All other applications run fine.  A full scan with Microsoft Defender finds no issues.

There have been no recent manual updates to system.  Windows and Chrome are set for auto-updating.

Please let me know if I may provide additional information.

Thank you.

Jeffrey

**Comment by ddennedy** [comment]
I do not understand "after approximately 85% clicks/actions." After what, or 85% of what? Maybe a word is missing.
Are you using preview scaling and proxies?
Did you wait for all waveforms and thumbnails to finish generating in the background?
What is Settings > Display Method set to?
The usual culprits are running low on memory or a rather long or complex project. That is a known weakness (that cannot be addressed simply through a bug report).

**Comment by bmatherly** [comment]
Sorry to ask, but have you tried to reboot?

**Comment by Zonenshine** [comment]
Dan,

Thank you for replying so quickly.  I am inserting answers below.

I do not understand "after approximately 85% clicks/actions." After what,
or 85% of what? Maybe a word is missing.
JZ: When interacting with the software one of course does so by clicking an
item (and related actions like keyboard shortcuts and dragging).  I am
estimating that out of a 100 interactions (clicks) 85 will trigger the Not
Responding condition which will last for about 30 seconds.
Are you using preview scaling and proxies?
JZ: I am not using proxies as the files are small.  Scaling is at 540p.
Did you wait for all waveforms and thumbnails to finish generating in the
background?
JZ: Yes
What is Settings > Display Method set to?
JZ: Automatic
The usual culprits are running low on memory or a rather long or complex
project. That is a known weakness (that cannot be addressed simply through
a bug report).
JZ: As noted, Task Manager indicates at least 60% of my 16 gigs of memory
are free.  The largest project is 6 minutes in length and I believe you
would not consider them very complex.

Thank you again for the willingness to help.  While my projects are small,
they are for my grandsons who look forward to them each week.

Jeffrey

**Comment by Zonenshine** [comment]
Brian,

Hi.  The question is reasonable and appreciated.  The problem has been
occurring for a few days now and I have rebooted many times.

All thoughts are appreciated.

Thank you,

Jeffrey

**Comment by ddennedy** [comment]
Can you please check in Task Manager > Details view if there is more than one shotcut.exe running?

Does the problem affect a new project - perhaps a scratch project just to test this - or only your existing project?

**Comment by Zonenshine** [comment]
Dan,

I believe I have identified the issue thanks to you pointing me in the
right direction.  As suggested, I checked Task Manager.  There was only one
shotcut.exe running but there was a second process under it:
crashpad_handler.exe.  It appears that crashpad_handler.exe is an error
handler associated with Google Drive and other apps.  I turned off Google
Drive and it appears Shotcut is again working fine.  I'm not sure why this
started happening since I only synch one unrelated directory to Google
Drive and I have not changed the Google Drive settings recently.  Until I
can find a more elegant solution, I will turn off Google Drive when using
Shotcut.

Thanks again for your help.

Jeffrey
